demos/python_demos/human_pose_estimation_3d_demo/modules/draw.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
`human_pose_estimation_3d_demo`'s modules are not supposed to be run separately. I suggest removing the line for all its modules.

Reply:
Yeah, I already removed it in my "shebang line fix" branch. I'll submit that change afterwards.